### ISSUE TITLE
How to know better evaluation metrics for Retrieval or Rank will translate into better actual recommendations in the case of MovieLens dataset?

### ISSUE THREAD

**Issue by houghtonweihu** [tensorflow/recommenders#691]
In the tutorials of Tensorflow Recommenders, top_k_categorical_accuracy is used for the evaluation of Retrieval, and mse for Rank. Do we have examples that show better evaluation metrics translate into better movie recommendations in the case of MovieLens dataset?

**Comment by caesarjuly** [comment]
@houghtonweihu I just came across your question. Hope my answer is not too late. In content recommendation system, AUC is almost the gold metric for evaluation, because rather than an accurate rating score, the ranking order is much more important. And in correct settings, higher AUC can always transfers to better production metrics, for example CTR. If sometimes you see AUC is increasing but CTR is worse or similar, then it should be some problems with either the training data or labels. For example, data leak means your features contain the information of labels, which will leads to a high AUC but bad performance when deploying.
Meanwhile, MovieLens dataset is way too small to considered as a real world modeling problem. You may try Criteo as alternative.
